services/venice-server/src/main/java/com/linkedin/venice/stats/ServerQuotaUsageStats.java

```diff
@@ -21,19 +20,19 @@ public class ServerQuotaUsageStats extends AbstractVeniceStats {
 
   public ServerQuotaUsageStats(MetricsRepository metricsRepository, String name) {
     super(metricsRepository, name);
-    requestedQPS = registerSensor("quota_rcu_requested", new Count());
-    requestedKPS = registerSensor("quota_rcu_requested_key", new Total());
-    rejectedQPS = registerSensor("quota_rcu_rejected", new Count());
-    rejectedKPS = registerSensor("quota_rcu_rejected_key", new Total());
-    allowedUnintentionallyKPS = registerSensor("quota_rcu_allowed_unintentionally", new Count());
+    requestedQPS = registerSensor("quota_requested_qps", new Count());
+    requestedKPS = registerSensor("quota_requested_kps", new Count());
+    rejectedQPS = registerSensor("quota_rejected_qps", new Count());
+    rejectedKPS = registerSensor("quota_rejected_kps", new Count());
+    allowedUnintentionallyKPS = registerSensor("quota_unintentionally_allowed_key_count", new Count());
```

Comment:
These metric names are a little confusing.
Can we mutate them in the following way?
```
quota_request
quota_request_key_count
quota_rejected_request
quota_rejected_key_count
````
I think we can use `Rate` metric type to make the semantics more explicit.
What do you think?


I assume these metrics cover all request types, right?

Reply:
Sure, will change them to the suggested names.

Yes, these metrics cover all request types (single, multi, compute, etc.).